packages/studio/src/components/AddPageButton/BasicPageDataCollector.tsx

```diff
@@ -41,6 +42,11 @@ export default function BasicPageDataCollector({
         const getPathValue = data.url
           ? createGetPathVal(data.url, isEntityPage)
           : undefined;
+        const pageDataValidator = new PageDataValidator();
+        pageDataValidator.validate(
+          { ...data, url: getPathValue?.value },
+          isEntityPage
```

Comment:
I'm thinking we should have `isEntityPage` as a variable to the `PageDataValidator()` constructor. It makes sense to me that we'd scope an instance of the validator to only Entity Pages or only Static Pages.